src/christinaGerdzhikova/week01/FINRA.java

```diff
@@ -23,5 +23,6 @@ public static void main(String[] args) {
 
         System.out.println(Finra(scan.nextInt()));
```

Comment:
This one should print out numbers 1 through 30 to look like this: 
1 2 FIN 4 RA FIN 7 8 FIN RA 11 FIN 13 14 FINRA, etc.

Reply:
fixed